### ISSUE TITLE
False positive message to delete the method, as the parent method is identical

### ISSUE THREAD

**Issue by Crovitche-1623** [kalessil/phpinspectionsea#1917]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended)  |
| Language level | PHP 8.2                       |

Current behaviour
--
A false positive message indicate to drop the method as it's identical to the parent one but it's not true as there is an additional PHP Attribute:
<img width="744" alt="image" src="https://github.com/kalessil/phpinspectionsea/assets/35468476/4d8e5678-b266-485a-be7f-8b05ac603132">

```php
<?php
// App\Entity\SlugTrait.php

trait SlugTrait
{
    // ...
    protected ?string $slug = null;

    public function getSlug(): ?string
    {
        return $this->slug;
    }
    // ...
}
```

Expected behaviour
--
Do not report this message if there is an additional / different PHP Attribute comparing to the parent method / method in trait.
